gateway-service/src/test/java/org/zowe/apiml/acceptance/corsTests/CorsPerServiceTest.java

```diff
@@ -47,6 +48,7 @@
 @AcceptanceTest
 @ActiveProfiles("test")
 @DirtiesContext
+@Disabled
```

Comment:
Why you want to disable this tests?

Reply:
Doesn't work on the older host environments such as Ubuntu 18.04, RHEL, and others

Reply:
Which also seems to be the case for the CircleCI. Maybe I can disable it under some profile instead of Disable in general

Reply:
I changed it to the DisabledIf, which allow us to disable on specific problematic environments while keeping the tests running on the non problematic ones.